ndb/src/google/cloud/ndb/exceptions.py

```diff
@@ -33,6 +34,20 @@ class Error(Exception):
     """Base datastore error type."""
 
 
+class AsyncContextError(Error):
+    """Indicates an async call being made without a context.
+
+    Raised whenever an asynchronous call is made outside of a context
+    established by :func:`google.cloud.ndb.async_context`.
```

Comment:
@chrisrossi For future reference, I typically use the `~` here, so that only the last part of the path is rendered.